plugins/src/main/java/org/smartregister/fhir/gateway/plugins/LocationHierarchyEndpointHelper.java

```diff
@@ -56,13 +59,18 @@ public LocationHierarchy getLocationHierarchy(
             String locationId,
             List<String> preFetchAdminLevels,
             List<String> postFetchAdminLevels,
-            Boolean filterInventory) {
+            Boolean filterInventory,
+            String lastUpdated) {
         LocationHierarchy locationHierarchy;
 
         if (CacheHelper.INSTANCE.skipCache()) {
             locationHierarchy =
                     getLocationHierarchyCore(
-                            locationId, preFetchAdminLevels, postFetchAdminLevels, filterInventory);
+                            locationId,
+                            preFetchAdminLevels,
+                            postFetchAdminLevels,
+                            filterInventory,
+                            lastUpdated);
         } else {
```

Comment:
Lets remove caching here for correctness then we can enhance with a new caching approach on a separate PR cc @dubdabasoduba